### PR TITLE
Integrate copy changes

### PR DESCRIPTION
**Pull request by rgarner**
**NO LONGER WIP, READY FOR REVIEW**

# [Integrate copy changes](https://trello.com/c/6aECcdOh/291-integrate-copy-changes)

screenshot-2017-12-7 preview print 12

### CODE REVIEW

**Comment by joelcarr** [comment]
Looks like we need to amend 'expenses' and 'incentives' sections at the bottom

**Comment by joelcarr** [comment]
@rgarner I suppose not.

File: app/views/research_sessions/preview.html.erb [hunk 1]
```diff
@@ -157,18 +136,112 @@
...
+    <p>
+      We will also ask <%= you_or_your_child %> if <%= you_or_they %> want to take
+      part before the session.
+    </p>
+    <p>
+      Don’t worry! If you do not want <%= 'the child in your care' if @research_session.unable_to_consent? %>
+      to take part it will not affect the service <%= you_or_they %> receive.
+    </p>
+    <p>
+      <%= you_or_they.capitalize %> can change your mind at any point for any
```
Comment (sampling) [inline app/views/research_sessions/preview.html.erb]: Currently it comes out as

>  They can change your mind at any point for any reason - just let us know!

Please change lines 194 and 195 to:

> *Your child / the child in your care* can change *their* mind during the session at any point for any reason by letting the researcher know *they* want to stop.
AND
> *You* can change *your* mind during the session at any point for any reason by letting the researcher know *you* want to stop.

File: app/views/research_sessions/preview.html.erb [hunk 2]
```diff
@@ -157,18 +136,112 @@
...
+      You can contact Jason Caplin to ask us to delete your personal data at any time.
+    </p>
+  </section>
+
+  <section>
+    <h3 class="subtitle-small" id="sharing-information">Sharing information</h3>
+    <p>
+      We may have to share information about <%= you_or_your_child %> without
+      your consent if we think <%= you_or_they %> or someone else is at risk
+      of serious harm. Where possible, the lead researcher speak to you first
```
Comment (sampling) [inline app/views/research_sessions/preview.html.erb]: "the lead researcher speak" should be changed to "the lead researcher will speak"

File: app/views/research_sessions/preview.html.erb [hunk 3]
```diff
@@ -157,18 +136,112 @@
...
-        for participation in this session.
-      </p>
-    </section>
-  <% end %>
+    <% end %>
+  </section>
+
+  <section>
+    <h3 class="subtitle-small" id="confidential">
+      Will anyone know what <%= i_or_they %> say in the discussion?
```
Comment (sampling) [inline app/views/research_sessions/preview.html.erb]: There should be no question mark at the end

File: app/views/research_sessions/preview.html.erb [hunk 4]
```diff
@@ -46,105 +46,84 @@
...
+              <dt><%= term %>:</dt>
+              <dd><%= edit_link_for(attr) %></dd>
+            </div>
+          <% end  %>
+        <% end %>
+      </dl>
       <p>
-        <%= edit_link_for(:methodologies) do %>
-          <%= methodology_lookup(methodology) %>
+        <%= your_or_your_childs.capitalize %> time and what <%= you_or_they %>
```
Comment (sampling) [inline app/views/research_sessions/preview.html.erb]: This sentence should only be visible when incentives enabled. If no incentive is provided, there's no point in saying that what you say is important..

File: app/views/research_sessions/preview.html.erb [hunk 5]
```diff
@@ -46,105 +46,84 @@
...
-    <h3 class="subtitle-small" id="what">What will happen next?</h3>
-
-    <% @research_session.methodologies.each do |methodology| %>
+    <h3 class="subtitle-small" id="session-details">Session details</h3>
+    <% if @research_session.where_when_enabled %>
+      <p>The session is</p>
+      <dl class="definition-list">
+        <%
+          {
+            when_text: 'On',
```
Comment (sampling) [inline app/views/research_sessions/preview.html.erb]: Change "On" to "When:"

File: app/views/research_sessions/preview.html.erb [hunk 6]
```diff
@@ -46,105 +46,84 @@
...
+            location:  'At',
```
Comment (sampling) [inline app/views/research_sessions/preview.html.erb]: Change "At" to "Where"

File: app/views/research_sessions/preview.html.erb [hunk 7]
```diff
@@ -46,105 +46,84 @@
...
     )
   %>
 
   <section>
-    <h3 class="subtitle-small" id="what">What will happen next?</h3>
-
-    <% @research_session.methodologies.each do |methodology| %>
+    <h3 class="subtitle-small" id="session-details">Session details</h3>
+    <% if @research_session.where_when_enabled %>
+      <p>The session is</p>
```
Comment (sampling) [inline app/views/research_sessions/preview.html.erb]: Please remove this line. We are happy with this section looking as:
> **Session details**
> When: 27th of September, 10AM
> Where: the National Museum
> Duration: 2 hours

File: app/views/research_sessions/preview.html.erb [hunk 8]
```diff
@@ -46,105 +46,84 @@
...
         prerender: true
       }
     )
   %>
 
   <section>
-    <h3 class="subtitle-small" id="what">What will happen next?</h3>
-
-    <% @research_session.methodologies.each do |methodology| %>
+    <h3 class="subtitle-small" id="session-details">Session details</h3>
```
Comment (sampling) [inline app/views/research_sessions/preview.html.erb]: If where/when is not enabled, we should not display the Session Details header (as it is followed by blank and then a next header)

File: app/models/recording_methods.rb [hunk 9]
```diff
@@ -1,12 +1,12 @@
...
-    workshop: 'Workshop outputs, such as drawings and post-its',
-    photo:    'Photographs',
-    screen:   'Screen recording',
-    other:    'Other'
+    voice:    'voice recording',
+    video:    'video recording',
+    written:  'written notes',
+    workshop: 'the work created by participants during the session',
+    photo:    'photos',
+    screen:   'a video of your screen as the participant uses the tool',
```
Comment (sampling) [inline app/models/recording_methods.rb]: Please change to "screen recording"

File: app/models/methodologies.rb [hunk 10]
```diff
@@ -1,11 +1,10 @@
 class Methodologies
   NAME_VALUES = {
-    interview:   'Interview',
-    usability:   'Usability testing',
-    survey:      'Survey',
-    focusgroup:  'Focus group',
-    codesign:    'Codesign workshop',
-    observation: 'Observation/field study',
+    interview:   'a one-on-one interview',
+    usability:   'an observation of trying out (the new tool) we’re designing',
```
Comment (sampling) [inline app/models/methodologies.rb]: Please remove the brackets. Expected outcome:
> an observation of trying out the new tool we’re designing

File: app/javascript/components/Shared/WhatHappensInThisResearchSession.js [hunk 11]
```diff
@@ -0,0 +1,41 @@
...
+      {Output(props, 'researcher_name')}
+      {jobTitle} would like {props.you_or_your_child} to take part in
+      {colonOrEllipsis}
+
+      <div dangerouslySetInnerHTML={{__html: props.methodologies_markup}} />
+      <div dangerouslySetInnerHTML={{__html: props.recording_markup}} />
+
+      <p>
+        There may be another researcher or observer supporting{' '}
+        {Output(props, 'researcher_name')}.
```
Comment (sampling) [inline app/javascript/components/Shared/WhatHappensInThisResearchSession.js]: Please add " at this session" at the end of the sentence to achieve:
> There may be another researcher or observer supporting Ida Rolek at this session.

File: app/views/research_sessions/preview.html.erb (hunk 1 above)
Comment (rgarner) [reply app/views/research_sessions/preview.html.erb]: I love that top error. Can we keep it?

File: app/views/research_sessions/preview.html.erb (hunk 3 above)
Comment (rgarner) [reply app/views/research_sessions/preview.html.erb]: @sampling Have to disagree. If it's a question it needs a question mark. We can remove the question mark if we rephrase it to not be a question.

File: app/models/methodologies.rb (hunk 10 above)
Comment (joelcarr) [reply app/models/methodologies.rb]: Can we safely assume it will always be a new tool?

File: app/views/research_sessions/preview.html.erb (hunk 8 above)
Comment (rgarner) [reply app/views/research_sessions/preview.html.erb]: This is slightly more complex – we've had to upgrade the logic around this section as there are three inter-relating parts that are mixed up together.

File: app/views/research_sessions/preview.html.erb [hunk 12]
```diff
@@ -198,7 +198,7 @@
       to take part it will not affect the service <%= you_or_they %> receive.
     </p>
     <p>
-      <%= you_or_they.capitalize %> can change your mind at any point for any
+      <%= you_or_your_child.capitalize %> can change <%= your_or_their %> mind at any point for any
```
Comment (sampling) [inline app/views/research_sessions/preview.html.erb]: The content here is not reflecting the commit message. It still reads:

> "mind at any point for any reason - just let us know!"
instead of 
> "during the session at any point for any reason by letting the researcher know **you / they**
want to stop."

Comment (rgarner) [reply app/views/research_sessions/preview.html.erb]: Good spot - rebased

**Comment by rgarner** [comment]
Two things were not right, we'd also not changed to "screen recording", now done.
